src/app.rs

```diff
@@ -124,6 +124,29 @@ fn run_ui<B: Backend>(
                             app.order_by_size();
                         }
                     }
+                    KeyCode::Char('d') => {
+                        if let Some(KeyCode::Char('a')) = app.last_keycode {
```

Comment:
for `da` keybinding, the current keycode should be `KeyCode::Char('a')` , the `app.last_keycode` should be `d`.

Reply:
fix it